docs/analyzers/FixAllProvider.md

```diff
@@ -117,4 +117,4 @@ Following guidelines should help in the implementation:
  - **GetSupportedFixAllScopes:** Virtual method to get all the supported FixAll scopes. By default, it returns all the three supported scopes: document, project and solution scopes. Generally, you need not override this method. However, you may do so if you wish to support a subset of these scopes.
  - **GetSupportedFixAllDiagnosticIds:** Virtual method to get all the fixable diagnostic ids. By default, it returns the underlying code fixer's `FixableDiagnosticIds`. Generally, you need not override this method. However, you may do so if you wish to support FixAll only for a subset of these ids.
 
-See [DeclarePublicAPIFix](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIFix.cs) for an example implementation of a custom FixAllProvider.
+See [DeclarePublicAPIFix](https://github.com/dotnet/roslyn-analyzers/blob/master/src/DotNetAnalyzers/Core/PublicApiAnalyzer.CodeFixes/DeclarePublicAPIFix.cs) for an example implementation of a custom FixAllProvider.
```

Comment:
```suggestion
See [DeclarePublicAPIFix](https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/Core/CodeFixes/DeclarePublicApiFix.cs) for an example implementation of a custom FixAllProvider.
```

Reply:
Thanks for you contribution. We have refactored the code in analyzer repo after you created this PR, so the link needs to be updated again :P